UnitTests/ConsoleDrivers/KeyTests.cs

```diff
@@ -167,25 +167,18 @@ public void Key_ToString ()
 		k = Key.Space;
 		Assert.Equal ("Space", k.ToString ());
 
-		k = Key.Space | Key.D;
-		Assert.Equal ("d", k.ToString ());
```

Comment:
You don't need to remove this because 'd' is equal to the bitwise `Key.Space | Key.D`, so 32+68=100, which is the ASCII 'd'.